src/Command/SnapshotTrait.php

```diff
@@ -95,6 +100,11 @@ protected function refreshDump(Arguments $args, ConsoleIo $io)
             $newArgs[] = $args->getOption('plugin');
         }
 
+        if ($args->getOption('source')) {
+            $newArgs[] = '-s';
+            $newArgs[] = $args->getOption('source');
+        }
```

Comment:
This is repeated quite a bit.

How about extracting it into a protected method?

``` php
$option = 'source';
$shortcut = '-s';
$this->addOptionToDump($args, $newArgs, $option, $shortcut);
```